src/components/publishing/article.tsx

```diff
@@ -23,6 +24,8 @@ interface ArticleState {
   article: any
 }
 
+// TODO: Determine where to implement Segment calls
+// @track({}, { dispatch: data => console.log(data) })
```

Comment:
good ol' console.log! 

I pushed some new things to my PR yday, we use the Emission/Eigen bridge to send events now and it [looks like this](https://github.com/artsy/emission/pull/693/files#diff-bb6e7220fc0d9b5c39f6b5b40352d143R22).